infra/label_evaluator_step_functions/lambdas/utils/tracing.py

```
@@ -0,0 +1,1215 @@
+            headers=run_tree.to_headers(),
+        )
+
+        logger.info("Trace started, yielding context (run_id=%s)", run_tree.id)
+        yield ctx
+
+        # End the run successfully
+        run_tree.end()
+        run_tree.patch()
+        logger.info("Trace ended and patched (run_id=%s)", run_tree.id)
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)